components/AboutMe/molecules/Social.tsx

```diff
@@ -4,10 +4,14 @@ import PsychoLink from 'components/atoms/PsychoLink';
 import SocialItem from 'components/AboutMe/atoms/SocialItem';
 import socialMediaItems from 'components/AboutMe/content/socialMedia';
 
+interface Props {
+  breakPoint: number;
+}
+
 const Container = styled.div`
   display: none;
 
-  @media screen and (min-width: 576px) {
+  @media screen and (min-width: ${({ breakPoint }: Props) => breakPoint}px) {
```

Comment:
Why do we do this? Is it because we are using this component on multiple pages?

Reply:
exactly, but 'Elérhetőségek' is longer than 'Rólam', so with 576 the content does not fit there 😄

Reply:
Honestly I don't have a better idea how to do this, but I feel like if you do something else for 6 motnhs then come back to the high level component, you might get confused about the breakPoint. Maybe we could add some extra documentation there, that it means the exact breakPoint in pixels? 😄

Reply:
does a comment qualify as documentation? 🤣